test/DynamoCoreWpfTests/InfoBubbleTests.cs

```diff
@@ -175,6 +176,7 @@ public void TestInfoState()
             Assert.NotNull(slider);
 
             DynamoModel model = GetModel();
+            (model.CurrentWorkspace as HomeWorkspaceModel).RunSettings.RunType = RunType.Manual;
```

Comment:
@reddyashish To preserve the test case, I need to convert Runtype to manual so that the graph execution triggered below would not remove the info. If we want to make the info to always show for integer slider, we probably need to introduce persistent info as new type just like persistent warning in code base

Reply:
Sounds good.